src/background/extension.js

```diff
@@ -103,39 +175,6 @@ export class Extension {
       }
     };
 
-    /**
-     * Initialize the extension.
-     *
-     * This queries the state of the extension in existing tabs and sets up
-     * event listeners to respond to future tab changes.
-     *
-     * If the extension's state in a particular tab cannot be determined,
-     * the extension is assumed not to be loaded in that tab.
-     *
-     * @return - A promise that resolves once listeners have been set up and
-     *   the state of existing tabs has been determined.
-     */
-    this.init = async () => {
```

Comment:
This function was moved down the file to avoid a reference-before-declaration error after changing some `function` expressions to arrow callbacks, to avoid confusion with `this` references.